packages/cbioportal-utils/src/mutation/MutationAnnotator.ts

```diff
@@ -290,17 +292,15 @@ export function annotateMutation(
 export function initAnnotatedMutation(
     mutation: Partial<Mutation>
 ): Partial<Mutation> {
-    const deepClone = toJS(mutation);
-
     return {
-        ...deepClone,
+        ...mutation,
```

Comment:
How do we handle typing of these extended mutations?   Should there be something called AnnotatedMutation?

Reply:
I don't think we introduce new fields when annotating a mutation. We just update/override existing `Mutation` model fields with annotated values. In this function we just set default (empty) values for those fields.